### PR TITLE
Fix tidy in t/ui/27-plugin_obs_rsync_status_details.t

### PR DESCRIPTION
**Pull request by andrii-suse**

### CODE REVIEW

File: docs/Contributing.asciidoc [hunk 1]
```diff
@@ -733,6 +733,16 @@ Script `.circleci/build_dependencies.sh` can be also modified when major
 changes are performed, e.g. different OS version or packages from forked OBS
 project, etc.
 
+Example commands to manually fix ci-packages.txt:
```
Comment (Martchus) [inline docs/Contributing.asciidoc:736]:
```suggestion
Example command to manually fix ci-packages.txt:
```

File: docs/Contributing.asciidoc [hunk 2]
```diff
@@ -733,6 +733,16 @@ Script `.circleci/build_dependencies.sh` can be also modified when major
...
+
+```
+# must be executed from root source folder
+docker run --rm -v$(pwd):/opt/testing_area registry.opensuse.org/devel/openqa/ci/containers/base:latest .circleci/build_dependencies.sh
```
Comment (Martchus) [inline docs/Contributing.asciidoc:740]:
```suggestion
docker run --rm -v"$PWD:/opt/testing_area registry.opensuse.org/devel/openqa/ci/containers/base:latest" .circleci/build_dependencies.sh
```

File: .circleci/build_cache.sh [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 set -ex
 
-sudo zypper ar -f -p 90 https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.2/openSUSE_Leap_15.2 openQA
+sudo zypper ar -f -p 90 http://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.2/openSUSE_Leap_15.2 openQA
```
Comment (Martchus) [inline .circleci/build_cache.sh:20]: That's not just a documentation change. Why do you change the protocol here?

**Review by okurz — CHANGES_REQUESTED** [review]
what @Martchus said

File: docs/Contributing.asciidoc (hunk 2 above)
Comment (Martchus) [reply docs/Contributing.asciidoc:740]: When running this command I get:

```
Problem: nothing provides perl(File::Map) needed by openQA-devel-4.6.1613477586.fa28c153e-lp152.3674.1.noarch
 Solution 1: do not install openQA-devel-4.6.1613477586.fa28c153e-lp152.3674.1.noarch
 Solution 2: break openQA-devel-4.6.1613477586.fa28c153e-lp152.3674.1.noarch by ignoring some of its dependencies
```

Does someone has to do some updating before?

**Comment by kalikiana** [comment]
I assume this is also obsolete now since #3726 included a commit with the fixes.

**Pull request closed without merging**